network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py

```diff
@@ -7,7 +7,7 @@
 @register.filter
 def yes_no(value):
     """Converts boolean to yes or no string"""
-    if value is None:
+    if value is None or value == '':
```

Comment:
In that case, we can probably just use `if not value:`, since both None and `''` are falsey, unlike `No` and `Yes`.